pom.xml

```diff
@@ -162,6 +185,16 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-gpg-plugin</artifactId>
```

Comment:
Can this be updated as well? 1.6 is from 2020, 3.2.4 is current. But perhaps this is out of scope for this PR.

Reply:
I'll try the new version first. So I'll draft this PR until that's done.